seolondon/settings/__init__.py

```diff
@@ -197,6 +200,10 @@
             'PORT': os.environ['RDS_PORT'],
         }
     }
+elif 'DATABASE_URL' in os.environ:
```

Comment:
Should we then amend the prod settings to use the same? Seems like we shoulf have it one way or the other really.